pyproject.toml

```diff
@@ -15,7 +15,7 @@ keywords = ["service", "infrastructure", "grpc"]
 readme = "README.md"
 # This license name comes from doc recommendation here
 # https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#license
-license = "LicenseRef-CognizantAcademicSource"
+license = "Apache-2.0"
```

Comment:
Please update the link to license docs as well.
It turns out the link we have now is stale.
This was called out in the leaf-common PR here: https://github.com/cognizant-ai-lab/leaf-common/pull/131#discussion_r2479041524